benchmark/benchmark.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+from sys import argv
+
+from engine import *
+
+
+def _path(x):
+    return os.path.join(os.path.dirname(__file__), '../%s' % x)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--engine_type', type=str, required=True)

```

Comment:
the default type is always str. do we need this here?

Reply:
no, we don't need it. I'll get rid of that.